lib/config.js

```diff
@@ -3,4 +3,13 @@
  */
 import {config} from '@bedrock/core';
 
-config.turnstile = {};
+config.turnstile = {
+  // account authorization type options
+  authorization: {
+    turnstile: {
+      // default secret testing key from turnstile
+      SECRET_KEY: '1x0000000000000000000000000000000AA',
+      url: 'https://challenges.cloudflare.com/turnstile/v0/siteverify'
+    }
+  },
```

Comment:
This module doesn't know where it will be used (it doesn't know anything about "accounts" or other target use cases). I'm not sure we need to have separate named use cases here and not just a single secret and URL. But, if we want to support multiple clients, I think we'd do it like this:

```suggestion
  clients: {
    // default client, expected to be used by most top-level apps;
    // other named clients can be added for more complex apps
    default: {
      // default secret testing key from turnstile
      SECRET_KEY: '1x0000000000000000000000000000000AA',
      url: 'https://challenges.cloudflare.com/turnstile/v0/siteverify'
    }
  }
```